### DISCUSSION TITLE
Error when accessing /admin on my webserver

### DISCUSSION THREAD

**Discussion by luciditylab** [filamentphp/filament#8291] [General]
As the title suggests, I am having issues with Filament on my webserver. Running the local server works as expected, but after deplying to my webserver I am receiving the following error when trying to access /admin

get_class(): Argument #1 ($object) must be of type object, string given

I am wondering if anyone has had a this issue before?

I am on a server provided by Digital Ocean using Plesk.
Laravel Version 10.22.0
PHP Version 8.2.9
OS: Ubuntu 20.04.6 LTS

**Comment by leandrocfe** [comment]
Is the composer ok?
`php artisan about`
what is the output?

**Reply by luciditylab** [reply]
Hi Leandro, thank you for your response. It turns out that there was a corrupetd file on my webserver. Not entirely sure how this occurred, but deleting all files and reuploading appears to have fixed the issue :)
